### ISSUE TITLE
Allow for defining disablePrefixOnAttributes and disableTypoToleranceOnAttributes

### ISSUE THREAD

**Issue by pixelastic** [algolia/algoliasearch-rails#193]
It seems that we cannot currently define [disablePrefixOnAttributes](https://www.algolia.com/doc/api-client/ruby/parameters/#disableprefixonattributes) nor [disableTypoToleranceOnAttributes](https://www.algolia.com/doc/api-client/ruby/parameters/#disabletypotoleranceonattributes) from the `algoliasearch do` block.

```ruby
  algoliasearch(index_name: "products") do
    disablePrefixOnAttributes ["code"]
    disableTypoToleranceOnAttributes ["code"]
  end
```

```
NoMethodError: undefined method `disablePrefixOnAttributes'
 for #AlgoliaSearch::IndexSettings:0x007f9d5682f0f0>
Did you mean?  disableTypoToleranceOnAttributes
from app/models/product.rb:106:in `block in <class:Product>`
```

Seems like there is a current whitelist of methods that can be called to define settings on the index. Shouldn't we make it agnostic so we can add new methods to the API without having to update all the clients when their values are straighforward?

![image](https://cloud.githubusercontent.com/assets/283419/22505008/69fd0ae4-e879-11e6-9e9e-0867f4c076d0.png)
https://github.com/algolia/algoliasearch-rails/blob/2d63c37ed6151e314e5479e6a102d1f48e23cd4f/lib/algoliasearch-rails.rb#L59-L69

cc @redox @jerskouille @marceldegraaf

**Comment by marceldegraaf** [comment]
@pixelastic thanks, I submitted a pull request for these two extra options.

**Comment by redox** [comment]
That's awesome guys. I'm merging those and release a new patch version of the gem!

**Comment by redox** [comment]
Released in `1.17.1` 👍  Thanks you your help @marceldegraaf
